src/Eloquent/HybridRelations.php

```diff
@@ -236,6 +236,11 @@ public function morphTo($name = null, $type = null, $id = null, $ownerKey = null
 
         $ownerKey ??= $instance->getKeyName();
```

Comment:
This line could go line 214 to avoid the elvis operator line 224.

Reply:
At line 214, The `$class` is `Null` and I can't create the instance to get its `key name`.